mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -164,6 +164,9 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 3 world
 		ARG 4 context
 		ARG 5 collisions
+	METHOD method_20802 getFireTime ()I
+	METHOD method_20803 setFireTime (I)V
```

Comment:
is this ticks? if yest, then set/getFireTicks

Reply:
The changed variable is called firetime so I just followed the name already in use, but yes it is ticks

Reply:
then rename the field to `fireTicks`, better than `fireTime` or `fire`

Reply:
https://github.com/FabricMC/yarn/pull/880/commits/9b33165535fd4e3ed55e6b4793cb2c6aa36e943f